### PR TITLE
PB-6066 | mysql deprecated args removed

### PR DESCRIPTION
**Pull request by ayush-px**
-> mysql deprecated args removed

### CODE REVIEW

File: charts/px-central/templates/px-lighthouse/px-central-ui/pxcentral-ui.yaml [hunk 1]
```diff
@@ -480,7 +480,6 @@ spec:
           image: {{ printf "%s/%s/%s:%s" .Values.images.mysqlImage.registry .Values.images.mysqlImage.repo .Values.images.mysqlImage.imageName .Values.images.mysqlImage.tag }}
           imagePullPolicy: {{ .Values.images.pullPolicy }}
           args:
-            - "--ignore-db-dir=lost+found"
```
Comment (kravichandran-px) [inline charts/px-central/templates/px-lighthouse/px-central-ui/pxcentral-ui.yaml:483]: @ayush-px Could you please share why this is not needed now, for my understanding.

File: charts/px-central/values.yaml [hunk 2]
```diff
@@ -196,7 +196,7 @@ images:
     registry: docker.io
     repo: portworx
     imageName: mysql
-    tag: 5.7.44
+    tag: 8.0.33
```
Comment (kravichandran-px) [inline charts/px-central/values.yaml:199]: Please push the new image to docker before merging this PR.

**Review by kravichandran-px — CHANGES_REQUESTED** [review]
@ayush-px Are we not planning to take a dump before upgrade?

**Comment by kravichandran-px** [comment]
You might need to run below as well:
```
cd stable                                                                                                                                                                                      
helm package ../charts/px-central
helm repo index --url https://raw.githubusercontent.com/portworx/helm/2.7.3-dev/stable/ .
```

File: charts/px-central/values.yaml (hunk 2 above)
Comment (kravichandran-px) [reply charts/px-central/values.yaml:199]: Same change needed in versions.yaml as well I hope.

Comment (ayush-px) [reply charts/px-central/values.yaml:199]: done

**Comment by px-kesavan** [comment]
@ayush-px Please update the raw_images_list.txt as well

**Comment by ayush-px** [comment]
> @ayush-px Please update the raw_images_list.txt as well

Done

**Review by px-kesavan — APPROVED** [review]
LGTM

**Pull request merged**